### PR TITLE
Use COM tlb instead of manually written interfaces

### PR DESCRIPTION
**Pull request by AArnott**

### CODE REVIEW

File: src/MSCorEE.Tests/MSCorEE.Tests.csproj [hunk 1]
```diff
@@ -17,12 +17,16 @@
...
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
+    <ResolveComReferenceSilent>true</ResolveComReferenceSilent>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
   <ItemGroup>
+    <COMFileReference Include="$(MSBuildProgramFiles32)\Windows Kits\NETFXSDK\4.6.1\Lib\um\x86\metahost.tlb">
```
Comment (hmemcpy) [inline src/MSCorEE.Tests/MSCorEE.Tests.csproj]: What is NETFXSDK a part of? Is it installed with Visual Studio?

**Comment by AArnott** [comment]
Yes, I believe all VS installations would have this.

File: src/MSCorEE.Tests/MSCorEE.Tests.csproj (hunk 1 above)
Comment (vbfox) [inline src/MSCorEE.Tests/MSCorEE.Tests.csproj]: It's also present in the sdk. If he can't be found in a specific location maybe we should put a copy in the repo

Comment (AArnott) [inline src/MSCorEE.Tests/MSCorEE.Tests.csproj]: We'd need to check the redist license if we put it in the repo, I *think*. Also as it's a binary, I'd rather just get it from the SDK on the machine -- at least until we find that a reasonably configured machine doesn't include it.

**Pull request merged**